_includes/v21.2/prod-deployment/healthy-sql-memory.md

```diff
@@ -0,0 +1 @@
+**Expected values for a healthy cluster**: This value should not exceed the [`--max-sql-memory`](recommended-production-settings.html#cache-and-sql-memory-size) size.
```

Comment:
I'm not sure this is correct or detailed enough - my understanding is that `--max-sql-memory` sets a soft limit and can be exceeded. But will that always result in a `memory budget exceeded` error, or does it happen more routinely?